library/openjdk

```diff
@@ -16,57 +16,52 @@ Architectures: amd64
 Architectures: windows-amd64
-GitCommit: 738932efd2236c905475115cf34f2a72cd65c02c
+GitCommit: e8d128ea8328ffb06d767e6210e952f4526ee6e8
 Directory: 14/jdk/windows/windowsservercore-1809
 Constraints: windowsservercore-1809
 
-Tags: 14-ea-8-jdk-windowsservercore-1803, 14-ea-8-windowsservercore-1803, 14-ea-jdk-windowsservercore-1803, 14-ea-windowsservercore-1803, 14-jdk-windowsservercore-1803, 14-windowsservercore-1803
-SharedTags: 14-ea-8-jdk-windowsservercore, 14-ea-8-windowsservercore, 14-ea-jdk-windowsservercore, 14-ea-windowsservercore, 14-jdk-windowsservercore, 14-windowsservercore, 14-ea-8-jdk, 14-ea-8, 14-ea-jdk, 14-ea, 14-jdk, 14
+Tags: 14-ea-9-jdk-windowsservercore-1803, 14-ea-9-windowsservercore-1803, 14-ea-jdk-windowsservercore-1803, 14-ea-windowsservercore-1803, 14-jdk-windowsservercore-1803, 14-windowsservercore-1803
+SharedTags: 14-ea-9-jdk-windowsservercore, 14-ea-9-windowsservercore, 14-ea-jdk-windowsservercore, 14-ea-windowsservercore, 14-jdk-windowsservercore, 14-windowsservercore, 14-ea-9-jdk, 14-ea-9, 14-ea-jdk, 14-ea, 14-jdk, 14
 Architectures: windows-amd64
-GitCommit: 738932efd2236c905475115cf34f2a72cd65c02c
+GitCommit: e8d128ea8328ffb06d767e6210e952f4526ee6e8
 Directory: 14/jdk/windows/windowsservercore-1803
 Constraints: windowsservercore-1803
 
-Tags: 14-ea-8-jdk-windowsservercore-ltsc2016, 14-ea-8-windowsservercore-ltsc2016, 14-ea-jdk-windowsservercore-ltsc2016, 14-ea-windowsservercore-ltsc2016, 14-jdk-windowsservercore-ltsc2016, 14-windowsservercore-ltsc2016
-SharedTags: 14-ea-8-jdk-windowsservercore, 14-ea-8-windowsservercore, 14-ea-jdk-windowsservercore, 14-ea-windowsservercore, 14-jdk-windowsservercore, 14-windowsservercore, 14-ea-8-jdk, 14-ea-8, 14-ea-jdk, 14-ea, 14-jdk, 14
+Tags: 14-ea-9-jdk-windowsservercore-ltsc2016, 14-ea-9-windowsservercore-ltsc2016, 14-ea-jdk-windowsservercore-ltsc2016, 14-ea-windowsservercore-ltsc2016, 14-jdk-windowsservercore-ltsc2016, 14-windowsservercore-ltsc2016
+SharedTags: 14-ea-9-jdk-windowsservercore, 14-ea-9-windowsservercore, 14-ea-jdk-windowsservercore, 14-ea-windowsservercore, 14-jdk-windowsservercore, 14-windowsservercore, 14-ea-9-jdk, 14-ea-9, 14-ea-jdk, 14-ea, 14-jdk, 14
 Architectures: windows-amd64
-GitCommit: 738932efd2236c905475115cf34f2a72cd65c02c
+GitCommit: e8d128ea8328ffb06d767e6210e952f4526ee6e8
 Directory: 14/jdk/windows/windowsservercore-ltsc2016
 Constraints: windowsservercore-ltsc2016
 
-Tags: 13-ea-32-jdk-oraclelinux7, 13-ea-32-oraclelinux7, 13-ea-jdk-oraclelinux7, 13-ea-oraclelinux7, 13-jdk-oraclelinux7, 13-oraclelinux7, 13-ea-32-jdk-oracle, 13-ea-32-oracle, 13-ea-jdk-oracle, 13-ea-oracle, 13-jdk-oracle, 13-oracle
-SharedTags: 13-ea-32-jdk, 13-ea-32, 13-ea-jdk, 13-ea, 13-jdk, 13
+Tags: 13-jdk-oraclelinux7, 13-oraclelinux7, 13-jdk-oracle, 13-oracle
+SharedTags: 13-jdk, 13
```

Comment:
This is the first official release candidate of OpenJDK 13 (which is also why Alpine is now gone); https://jdk.java.net/13/:

> The Alpine Linux build previously available on this page was removed as of the first JDK 13  release candidate. It’s not production-ready because it hasn’t been tested thoroughly enough to be considered a GA build. Please use the early-access JDK 14 Alpine Linux build in its place.